layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -100,11 +111,174 @@
   <ul class="list-unstyled" id="search-results"></ul>
   <ul class="list-unstyled" id="default-body">
     {{ range $key, $value := $registry -}}
-      {{ $value = merge $value (dict "_key" $key) -}}
-      {{ partial "ecosystem/registry/entry" (dict "value" $value "languageNames" $languageNames) }}
+    {{ $value = merge $value (dict "_key" $key) -}}
+    {{ partial "ecosystem/registry/entry" (dict "value" $value "languageNames" $languageNames) }}
     {{ end -}}
   </ul>
+  <!-- Pagination Controls -->
+  <div id="pagination" class="text-center">
+    <button id="prev-page" class="btn btn-outline-secondary">Previous</button>
+    <div id="pagination-numbers" class="d-inline-block"></div>
+    <button id="next-page" class="btn btn-outline-secondary">Next</button>
```

Comment:
make sure that there is some spacing between the Previous, the count of pages, and the next